### ISSUE TITLE
Ctrl+K doesn't work in the multiline input box.

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#1347]
**Reported by Andrio on 10 Dec 2012 05:49:31 UTC**
Currently there seems to be no way to input colour codes into the multiline input box, other than copying and pasting the raw control character. Pressing Ctrl+K inputs $k, but that doesn't do anything. Alt+03 doesn't do anything either.
Also, $b and $o don't work.

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/1347

**Comment by un1versal** [comment]
Confirmed this is still a bug in current dev builds.

**Comment by un1versal** [comment]
@AndrioCelos Well, this is KVS so $k is the KVS color control character, you need to add the color by numbers after. but it would be nice to get a colors popup to know which ones are which.

Changed this to feature request, since this is normal behaviors for KVS afaik.

**Comment by AndrioCelos** [comment]
Did you try it?
Your entry is only treated as KVS if it begins with a `/` **and** user-friendly mode is off.

**Comment by un1versal** [comment]
yes, with friendly mode on or off at the  very beginning of the multi-line input (from empty) pressed ctrl+k and as expected $k is entered...

idk what else is supposed to do.

**Comment by un1versal** [comment]
what doesnt work is ctrl+i all other control characters in KVS are entered by pushing respective shortcuts

**Comment by un1versal** [comment]
Ive opened https://github.com/kvirc/KVIrc/issues/2011 because KVS should not require / before a command... that to me a a bug or faulty implementation

Im not sure what the best thing is either for this report, but it would be nice if you had a popup with the colors same as you do in normal input editor... and added a braket soon as you press `Ctrl+k` you get **`$k()`** since the color in KVS must be in between parenthesis also.

**Comment by un1versal** [comment]
what I can do for this is add make sure that `$k()` is output when pressing `ctrl+k` instead of just `$k`, because colors in KVS need to be surrounded by parenthesis as you know in `$k(color)` or $k(color,color) format...

else Im not sure much else can be done here, the bug here is really --> https://github.com/kvirc/KVIrc/issues/2011

**Comment by IceN9ne** [comment]
I'd be in favor of changing ctrl+&lt;key&gt; to be the normal control code instead of $&lt;key&gt;. You're not saving any substantial key strokes and it's more natural for it to be the control codes instead of the KVIrc function. This change would allow you to easily use either method in the box.

Replying to the $k with parentheses issue, a plain $k will reset the color back to default, so it shouldn't by default add the parentheses.
